package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@themetadao/futarchy-ts",
-  "version": "1.0.5-alpha.1",
+  "name": "@themetadao/futarchy-sdk",
```

Comment:
I think we are changing the org name to as well? To like @metaDAOProject

Reply:
Let's change the scope in a future PR. Not seeing a clear consensus on this right now
https://discord.com/channels/1155877543174475859/1174824703513342082/1233284981833469952